core/ui.js

```diff
@@ -213,7 +215,16 @@ async function addIconIfNeeded()
 
 function generateDropContent(options)
 {
-    var presenceCaption = "Will prevent you from seeing other people's last seen and typing status";
+    var onlineStatusTitle = "Hide \"Online\" status";
+    var onlineStatusCaption = "Stops sending presence updates. If enabled, will prevent you from seeing others' online status.";
+
+    var typingStatusTitle = "Hide \"Typing...\" status";
+    var typingStatusCaption = "Will prevent you from sending 'Typing...' status.";
```

Comment:
Actually the "will you prevent you" here also sounds weird. Let's just say "Stops sending composing updates"

Reply:
Done